### PR TITLE
[6091] Expiry/Expired date shown in "Expiring" dashlet is off by 4 hours

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/6091

### CODE REVIEW

File: static-assets/components/cstudio-forms/controls/date-time.js [hunk 1]
```diff
@@ -43,7 +43,11 @@ CStudioForms.Controls.DateTime =
     this.startDateTimeObj = null; // Object storing the time when the form was loaded; will be used to adjust startTzDateTimeStr before the form is saved
     this.startTzDateTimeStr = null; // Time the form was loaded (adjusted to the site's timezone)
     this.populateDateExp = 'now';
-    this.defaultTimezone = 'UTC';
+    // This value is used when no custom timezone is selected for the control, the defaultTimezone is used for the timezone
+    // this will actually get stored, defaults to user timezone.
+    this.displayTimezone = craftercms.utils.datetime.getUserTimeZone();
+    // this.defaultTimezone = 'UTC';
```
Comment (rart) [inline static-assets/components/cstudio-forms/controls/date-time.js]: Did you mean to delete?

File: static-assets/components/cstudio-forms/controls/time.js [hunk 2]
```diff
@@ -38,7 +38,11 @@ CStudioForms.Controls.Time =
     this.startDateTimeObj = null; // Object storing the time when the form was loaded; will be used to adjust startTzDateTimeStr before the form is saved
     this.startTzDateTimeStr = null; // Time the form was loaded (adjusted to the site's timezone)
     this.populateDateExp = 'now';
-    this.defaultTimezone = 'UTC';
+    // This value is used when no custom timezone is selected for the control, the defaultTimezone is used for the timezone
+    // this will actually get stored, defaults to user timezone.
+    this.displayTimezone = craftercms.utils.datetime.getUserTimeZone();
+    // this.defaultTimezone = 'UTC';
```
Comment (rart) [inline static-assets/components/cstudio-forms/controls/time.js]: Delet?

**Review by rart — CHANGES_REQUESTED** [review]

File: static-assets/components/cstudio-forms/controls/date-time.js (hunk 1 above)
Comment (jvega190) [reply static-assets/components/cstudio-forms/controls/date-time.js]: Yeah, deleting

File: static-assets/components/cstudio-forms/controls/time.js (hunk 2 above)
Comment (jvega190) [reply static-assets/components/cstudio-forms/controls/time.js]: Yeah, deleting

**Review by rart — APPROVED** [review]

**Pull request merged**